package/bleparser/almendo.py

```diff
@@ -0,0 +1,76 @@
+"""Parser for Almendo bluSensor BLE advertisements"""
+# black -l 79
+import logging
+from struct import unpack
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def parse_almendo(self, data, source_mac, rssi):
+    """Almendo parser"""
+    result = {
+        "mac": "".join(f"{x:02X}" for x in source_mac[:]),
+        "type": "Almendo",
```

Comment:
Suggestion: remove this line. 

The `type` should represent the device type, so in your case the `Mini BSP02AIQ`
The `firmware` should represent the company, so either `Almendo` or `BluSensor`. If there are different versions of the firmware, you can add V1. I'm not sure about the name Almendo. If I check the website of the sensor, I only see BlueSensor, so why did you use Almendo? Even the URL is called blusensor.com. https://www.blusensor.com/en/products/

Reply:
Almendo seems to have a number of different devices using the 0x06E8 manufacturer specific data tag. In that data is a byte describing which hardware model it is (0x0A for BSP02AIQ, and 0x09 for BSP02AIR, unknown what they use for BSP01AIR; the documentation is a bit spotty), and one byte describing the protocol version. The BSP02AIR lack a few of the sensors of BSP02AIQ, and I guess, but can't know for sure, that they do not include data for the missing sensors in the data from that device.

The bottom line is that I suspect that there could be a lot of firmware versions using the same protocol version, because they chose not to tag the data itself.